app/src/debug/res/layout/contactpatient_callpatient.xml

```diff
@@ -0,0 +1,347 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
In some places, we are referencing the dimension values and some are hardcoded. Let's make that consistent. Can be done in next PR.